scripts/action-recognition/test_recognizer.py

```diff
@@ -244,7 +244,7 @@ def calibration(net, val_data, opt, ctx, logger):
     if isinstance(ctx, list):
         ctx = ctx[0]
     exclude_sym_layer = []
-    exclude_match_layer = []
+    exclude_match_layer = ['flatten']
```

Comment:
why exclude `flatten` here.

Reply:
After double check, looks there is no need to explicitly specify `flatten` as being excluded.